example/lib/pages_with_tiled_images/main_page_list_images_inspector.dart

```diff
@@ -34,7 +34,7 @@ class PageListImagesInspectorDemo extends StatefulWidget {
 }
 
 class _PageListImagesInspectorDemoState extends State<PageListImagesInspectorDemo> with TickerProviderStateMixin {
-  static final _naturalPageSize = const Size(8.5, 11) * 72;
+  static final _naturalPageSize = const Size(8.5, 11) * 14.4;
```

Comment:
This still will not decode the images at a smaller size.

You need to find all the places where you're using an `AssetImage` and wrap it with a `ResizeImage`.

Reply:
I only found `AssetImage` being used [here](https://github.com/Flutter-Bounty-Hunters/page_list_viewport/blob/main/example/lib/pages_with_tiled_images/image_cache.dart#L477) but I don't quite see how to use `ResizeImage` there.

Reply:
Oh, I figured it out.